aip/general/0131.md

```diff
@@ -83,24 +83,21 @@ variable in the `google.api.http` annotation on the RPC. This causes the `name`
 field in the request to be populated based on the value in the URL when the
 REST/JSON interface is used.
 
-[aip-121]: ./0121.md
-[aip-123]: ./0123.md
-[aip-157]: ./0157.md
-[aip-203]: ./0203.md
 
 ### Errors
 
-If the user does not have permission to access the resource, regardless of
-whether or not it exists, the service **must** error with `PERMISSION_DENIED`
-(HTTP 403). Permission **must** be checked prior to checking if the resource
-exists.
+See [errors][].
```

Comment:
You are moving specific section of error handling, perhaps you can link directly to that section and provide related statement in the referencing docs. ex: See "Permission Denied" section in the AIP-193.

Reply:
ack. I wonder if we can just remove the section entirely?

We could just link to permission denied, but the whole of the error guidance is more critical. For example, the standard [payloads include a set of critical requirements on the response returned](https://google.aip.dev/193#guidance).

Reply:
Yes, we could remove the section as long as long as the linking to 193 is properly emphasized.

Also if we are considering the "permission denied" section being the most pertinent for the context
it should also be emphasized even if we are recommending for the reader to review the entire AIP-193.

Reply:
What's your suggestion? I could put this in "further reading"?

Reply:
I don't have an example to point to so for the time being I think keeping a section header would probably be the best unless we come up with an acceptable pattern.

Reply:
ack. Added a specific reference. PTAL!